protobuf/check_custom_value.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package Trento.Checks.V1;
+option go_package = "/;events";
+
+message CheckCustomValue {
+    string name = 1;
+    string value = 2;
```

Comment:
question: is the value always a string? 
We accept the string, numbers and boolean. Shouldn't we send the appropriate type? Otherwise we will save the metadata always as string, and after all, we can use the metadata for searching for example. I think using the correct type is valuable

Reply:
yes, totally, thanks for pointing out. Gonna fix it.